lib/components/app/app-nav.js

```diff
@@ -4,26 +4,33 @@ import { connect } from 'react-redux'
 
 import NavLoginButtonAuth0 from '../user/nav-login-button-auth0.js'
 import { accountLinks, getAuth0Config } from '../../util/auth'
-
 import AppMenu from './app-menu'
 
-const AppNav = ({ otpConfig }) => {
+const AppNav = ({ otpConfig, title }) => {
   const { branding, persistence } = otpConfig
   const showLogin = Boolean(getAuth0Config(persistence))
 
   return (
     <Navbar fluid inverse>
       <Navbar.Header>
         <Navbar.Brand>
-          <div className='app-menu-container'>
+          {/* TODO: Reconcile CSS class and inline style. */}
+          <div className='app-menu-container' style={{ float: 'left', color: 'white', fontSize: 28 }}>
             <AppMenu />
           </div>
-          <div
-            className={`icon-${branding}`}
-            // This style is applied here because it is only intended for
-            // desktop view.
-            style={{ marginLeft: 50 }}
-          />
+
+          {branding && (
+            <div
+              className={`icon-${branding}`}
+              // This style is applied here because it is only intended for
+              // desktop view.
+              style={{ marginLeft: 50 }}
+            />
+          )}
+
+          {title && (
+            <div className='navbar-title' style={{ marginLeft: 50 }}>OpenTripPlanner</div>
```

Comment:
`title` prop is not actually used? Move "OpenTripPlanner" to default prop perhaps?

Reply:
Oops!